sql/core/src/main/scala/org/apache/spark/sql/jdbc/MySQLDialect.scala

```diff
@@ -210,6 +210,7 @@ private case class MySQLDialect() extends JdbcDialect with SQLConfHelper {
     case FloatType => Option(JdbcType("FLOAT", java.sql.Types.FLOAT))
     case StringType => Option(JdbcType("LONGTEXT", java.sql.Types.LONGVARCHAR))
     case ByteType => Option(JdbcType("TINYINT", java.sql.Types.TINYINT))
+    case TimestampNTZType => Option(JdbcType("DATETIME", java.sql.Types.TIMESTAMP))
     case _ => JdbcUtils.getCommonJDBCType(dt)
```

Comment:
for my own education, `JdbcUtils.getCommonJDBCType` does not work for NTZ?

Reply:
Partially, the typeName plays an important role in timestamp mapping but  `JdbcUtils.getCommonJDBCType` only cares about java.sql.Types.

Reply:
Then...
https://github.com/apache/spark/blob/5afb2d14c24557352e2e21996028d44416e3d208/sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/jdbc/JdbcUtils.scala#L153-L157